dolphinscheduler-data-quality/pom.xml

```diff
@@ -171,34 +171,4 @@
         </dependency>
 
     </dependencies>
-
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-assembly-plugin</artifactId>
-                <version>2.2</version>
-                <configuration>
-                    <appendAssemblyId>false</appendAssemblyId>
-                    <descriptorRefs>
-                        <descriptorRef>jar-with-dependencies</descriptorRef>
-                    </descriptorRefs>
-                    <archive>
-                        <manifest>
-                            <mainClass>org.apache.dolphinscheduler.data.quality.DataQualityApplication</mainClass>
-                        </manifest>
-                    </archive>
-                </configuration>
-                <executions>
-                    <execution>
-                        <id>make-assembly</id>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>single</goal>
```

Comment:
I remember that the developer added this to improve the packaging speed. Will it have any effect after removing it?
cc @zixi0825

Reply:
It doesn't make sense to me `jar-with-dependencies` packaging will improve the packaging speed, please provide more details @zixi0825

Reply:
Jar-with-dependencies has nothing to do with improving packaging speed. I only use this to make the data quality task easier to use. When running the spark application, there is no need to add the paths of various drivers to reduce errors. I will mention PR to remove jar-with-dependencies.

Reply:
> Jar-with-dependencies has nothing to do with improving packaging speed. I only use this to make the data quality task easier to use. When running the spark application, there is no need to add the paths of various drivers to reduce errors. I will mention PR to remove jar-with-dependencies.

@zixi0825 Thank you, that make sense to me. How do you plan to remove jar-with-dependencies, will that effect how users use this module?

Reply:
> My idea is to add an attribute to the configuration file, fill in the list of jars that need to be used (provide default values), read the attribute and automatically add the configuration when running the Spark application, and it should have no effect on the user.

Reply:
> > Jar-with-dependencies has nothing to do with improving packaging speed. I only use this to make the data quality task easier to use. When running the spark application, there is no need to add the paths of various drivers to reduce errors. I will mention PR to remove jar-with-dependencies.
> 
> @zixi0825 Thank you, that make sense to me. How do you plan to remove jar-with-dependencies, will that effect how users use this module?

My idea is to add an attribute to the configuration file, fill in the list of jars that need to be used (provide default values), read the attribute and automatically add the configuration when running the Spark application, and it should have no effect on the user.